### ISSUE TITLE
nn.functional.layer_norm输出与pytorch不一致

### ISSUE THREAD

**Issue by huangzheng86** [Oneflow-Inc/oneflow#10306]
### Description

以下代码，在传入参数相同的情况下，oneflow和pytorch输出结果不一致
import oneflow as torch
with torch.autocast(enabled=False, device_type=module_device.type):
    return torch.nn.functional.layer_norm(downcast_x, self.normalized_shape, downcast_weight, downcast_bias, self.eps)

**Comment by shicheng0829** [comment]
试了下oneflow目前LayerNorm反向传播的梯度和pytorch的结果对不上，可以用以下代码检验

```python
import torch
dtype = torch.float
batch_size, in_features = 16, 2

x = torch.randn(batch_size, in_features, dtype=dtype, device='cuda:0')
x.requires_grad = True
layer_norm = torch.nn.LayerNorm(normalized_shape=[in_features,], device='cuda:0', dtype=dtype)
y = layer_norm(x)
x.retain_grad()
torch.sum(y).backward()


print(layer_norm.weight.grad)
print(layer_norm.bias.grad)
print(x.grad)


import oneflow as flow
import oneflow.nn as nn
class MyModel(nn.Module):
    def __init__(self):
        super(MyModel, self).__init__()
        self.layer_norm = nn.LayerNorm((in_features,))  # Replace 'input_size' with the actual size of the input

    def forward(self, x):
        x = self.layer_norm(x)
        return x

# 创建模型实例
model = MyModel()

# 创建输入数据
input_data = flow.Tensor(x.cpu().detach().numpy())  # 用实际的输入数据替换'...'
print(x)

# 启用Autograd，跟踪输入梯度
input_data.requires_grad = True

# 运行前向传播
output = model(input_data)

# 执行反向传播，计算梯度
output.sum().backward()

# 现在，input_data.grad将包含LayerNorm层输入的梯度
print(input_data.grad)

```
